src/runtime/node/buffer/index.ts

```diff
@@ -22,8 +22,8 @@ export const transcode = notImplemented("buffer.transcode");
 export const isUtf8 = notImplemented("buffer.isUtf8");
 export const isAscii = notImplemented("buffer.isAscii");
 
-export const btoa = global.btoa;
-export const atob = globalThis.atob;
+export const btoa = global.btoa.bind(global);
```

Comment:
For consistency:

```suggestion
export const btoa = globalThis.btoa.bind(globalThis);
```

But because this is a fix it should better be in a different PR (together with the test).

Reply:
One of the tests are failing because workerd throws an error for `invalid this`. I'll apply your suggestion. Thanks for the review.